src/ibek/ioc_cmds/assets.py

```diff
@@ -57,14 +63,24 @@ def extract_assets(
             GLOBALS.PVI_DEFS,
             GLOBALS.IBEK_DEFS,
             IOC_FOLDER,
-            Path("/venv"),
```

Comment:
Idea:
- remove venv from asset_matches
- remove ibek from epics-base
- make move assets in epics-base manual
- add pip install -r requirements.txt in runtime stage of IOCs
- then this guarentees same version in runtime/developer/proxy !!!

Reply:
@coretl there is a small downside to this genius idea.
Someone has to create the venv in developer and runtime.
At present the venv is created in epics-base developer and copied across to epic-base runtime and thus is available to all ioc stages (except proxy ones that also create a venv).

If we go with this solution then each stage in every IOC will need to 

- install python3-venv
- python3 -mvenv /venv
- ENV PATH=/venv/bin:${EPICS_BASE}/bin/${EPICS_HOST_ARCH}:${PATH}

We could of course do this in base and just not put anything in it.
We could also do away with the venv altogether since we are no longer copying it around.

BUT: rtems-proxy as a python-copier project has a venv. When I tried unpicking this on Sunday I found it was a little harder than expected (especially setuptools_scm getting upset)

Reply:
I've removed ibek from epics-base https://github.com/epics-containers/epics-base/commit/a76fd1963832bb57811575780fb4d27b5c8ab219

Reply:
Minor niggle:
I had to install ca_certificates and python3_disutils into epics-base runtime target so that pip would work (because I'm using python3-minimal)